pelican/settings.py

```diff
@@ -90,11 +90,11 @@ def load_source(name, path):
         (1, '{name}{extension}', '{name}{extension}'),
         (2, '{name}{number}{extension}', '{name}{number}{extension}'),
     ],
-    'YEAR_ARCHIVE_URL': '',
+    'YEAR_ARCHIVE_URL': 'posts/{date:%Y}/',
     'YEAR_ARCHIVE_SAVE_AS': '',
-    'MONTH_ARCHIVE_URL': '',
+    'MONTH_ARCHIVE_URL': 'posts/{date:%Y}/{date:%b}/',
     'MONTH_ARCHIVE_SAVE_AS': '',
-    'DAY_ARCHIVE_URL': '',
+    'DAY_ARCHIVE_URL': 'posts/{date:%Y}/{date:%b}/{date:%d}/',
```

Comment:
Why the default values?

Reply:
Mainly just to provide sensible defaults (that were already suggested elsewhere in the settings docs before this PR) to help a bit with configuring new Pelican sites. (With this PR, if users choose to enable period archives, they also need to configure the URLs, unlike before when those settings were only used for `PAGINATION_PATTERNS`).

I can see now though that having defaults could be more confusing, especially since the `*_ARCHIVE_SAVE_AS` settings should match.

Should I revert to blank defaults and just edit the docs accordingly?

Reply:
I believe so. These defaults won't do anything without `*_SAVE_AS` being defined as well. I think it would be confusing.

Reply:
I've pushed a commit to revert to blank defaults.